near-mock-vm/assembly/__tests__/main.ts

```diff
@@ -1,6 +1,6 @@
 export function add(a: i32, b: i32): i32 {
   for (let i = 0; i < a * b; i++) {
-    let x = i + a + b;
+    let _x = i + a + b;
```

Comment:
Do you need the `let _x =` here? Why not:

```ts
for (let i = 0; i < a * b; i++) {
  i + a + b;
}
```

Reply:
It was to test the ignoring of unused vars.